extensions/flyway/deployment/src/test/java/io/quarkus/flyway/test/FlywayExtensionCleanAtStartTest.java

```diff
@@ -48,6 +48,8 @@ public void testFlywayConfigInjection() throws SQLException {
             } catch (JdbcSQLException e) {
                 // expected fake_existing_tbl does not exist
             }
+        } finally {
+            connection.close();
```

Comment:
Maybe we should use a try with resources for this, too?

Reply:
ok, I will change it to this approach: `try (Connection connection = defaultDataSource.getConnection(); Statement stat = connection.createStatement()) {`